components/shared/filters/MobileFilter.vue

```diff
@@ -10,12 +10,13 @@
     class="top is-absolute background-color">
     <div class="is-flex is-flex-direction-column is-fullheight">
       <div class="is-flex-grow-1">
-        <div class="is-flex border-bottom">
-          <p class="card-header-title has-text-weight-bold">
+        <div
+          class="filters-header border-bottom is-flex is-align-items-center is-justify-content-space-between">
+          <p class="has-text-weight-bold">
             {{ $t('general.filters') }}
           </p>
-          <a class="card-header-icon">
-            <NeoIcon icon="x" @click="onClose" />
+          <a>
+            <NeoIcon icon="xmark" @click="onClose" />
```

Comment:
**suggestion**: put click event on `button` or `a` tag not on icon

Reply:
fixed